pkg/cmd/pulumi/org.go

```diff
@@ -123,6 +126,64 @@ func newOrgSetDefaultCmd() *cobra.Command {
 	return cmd
 }
 
+func newOrgListCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "ls",
+		Short: "List the orgs available to the user for the current backend",
+		Long: "List the orgs available to the user for the current backend.\n" +
+			"\n" +
+			"This command is used to list all of the organizations the user is a part of within " +
+			"the current backend.\n" +
+			"\n" +
+			"Currently, only the managed and self-hosted backends support organizations.",
```

Comment:
Should we add something along the lines of "self-managed backends like AWS S3 or Azure Blob Storage do not support organisations" to help distinguishing between self-hosted and self-managed? I don't feel super strongly about this, but I was a little confused at first.

Reply:
Sure, I can modify but we should probably update all of these since I grabbed this block from the set default command

Reply:
Is DIY the public name or just in the code? The docs still seem to refer to "self-managed", https://www.pulumi.com/docs/iac/concepts/state-and-backends/#using-a-self-managed-backend

Reply:
Docs should have been updated to match

Reply:
Updated the docstring @julienp @Frassle

Reply:
`If you try and set a default organization for a backend that does not support create organizations, then an error will be returned by the CLI`

The `does not support create organizations` part sounds weird to me. I realise this is copied from the existing code. The first part is also specific to the operation in question. Should we just keep the `DIY backends (e.g. AWS S3 or Azure Blob Storage) do not support organizations."` bit and drop the rest?